src/games/ecards.py

```diff
@@ -0,0 +1,275 @@
+        Reads in prompts, puts the deck in a particular state, and runs the game.
+        """
+
+        # check if prompts
+        if self.prompts == self.safeties == None:
+            # info message
+            self.logging.info(f"Reading prompts and safety files for {self.guild.id}.")
+            await self.channel.send(
+                embed=self.embed(f"Reading prompts and safety files...")
+            )
+
+            # read files
+            try:
+                self.prompts = self._read_file(self.file_prompts)
+                self.safeties = self._read_file(self.file_safeties)
+            except Exception as e:
+                self.logging.error(f"Error reading files {e}")
+                # early exit
+                return await self.channel.send(
+                    embed=self.embed(f"Error reading files\n```\n{e}\n```")
+                )
+
+            self.logging.info(
+                f"Read in {len(self.prompts)} prompts and {len(self.safeties)} safeties."
+            )
+
+        # create deck orders
+        prompt_deck = random.sample(self.prompts, len(self.prompts))
+        answer_deck = random.sample(self.safeties, len(self.safeties))
+
+        # create starting hands
+        hands = {pid: [answer_deck.pop() for _ in range(5)] for pid in self.players}
+
+        # run a round for each player
+        for pid in self.players:
+            self.logging.info(
+                f"new ecards round with leader player {self.players[pid]}"
+            )
+            await self.execute_round(pid, prompt_deck.pop(), hands)
+            ECards.refill_hands(hands, answer_deck)
+            asyncio.sleep(self.wait_duration)
+        return await self.scoreboard()
+
+    @staticmethod
+    def refill_hands(hands: dict, answer_deck: list):
+        """
+        Refills the players' hands from a deck of answers after a round.
+        Objects are passed by reference and modifications made in place, hence no need to return.
+
+        :param hands: a dictionary mapping player index to player hand
+        :param answer_deck: a deck of answer cards to fill hands with
+        """
+        for key in hands:
+            while len(hands[key]) < 5:
+                hands[key].append(answer_deck.pop())
+
+    async def execute_round(self, leader: str, prompt: str, hands: dict):
+        """Executes exactly one round of the game.
+
+        This involves
+        - sending the prompt to all players
+        - allowing non-leader players to select a card
+        - presenting all answers to the leader - allowing them to vote
+        - displaying round results to channel
+
+        :param leader: the index of the leader player
+        :param prompt: the prompt for the round
+        :param hands: the players' hands
+        """
+        # announce new round
+        await self.channel.send(
+            embed=self.embed(
+                f"Starting new round -- {self.players[leader]} is leader.\nThis round's prompt: {prompt}"
```

Comment:
Can you wrap `{prompt}` in bold, i.e. `**{prompt}**`? Makes it easier to read in the output :)